Marlin/src/Marlin.cpp

```diff
@@ -1131,7 +1149,14 @@ void loop() {
         #if DISABLED(SD_ABORT_NO_COOLDOWN)
           thermalManager.disable_all_heaters();
         #endif
-        thermalManager.zero_fan_speeds();
```

Comment:
Shouldn't this line persist; such that in a combined extruder/cutter system the fans properly power down?

Reply:
Yea, you're right. I just wanted to make sure the fan pin doesn't accidentally trigger laser/spindle for any reason.